app/views/talking_stick/rooms/show.html.erb

```diff
@@ -107,7 +107,7 @@ function isInt(value){
 
 $(window).on('talking_stick.new_partner', function(ev, partner) {
   $('#talking-stick-partnervideos').append(partner.videoElement);
-};
+});
```

Comment:
You should probably cherry-pick this to the `develop` branch since it's unrelated to Rails 5

Reply:
Will do over the weekend.